replicatelogs.go

```diff
@@ -624,3 +604,86 @@ func peakBaggedRoot(state massifs.MMRState) []byte {
 	}
 	return mmr.HashPeaksRHS(sha256.New(), state.Peaks)
 }
+
+type changeCollector struct {
+	log         logger.Logger
+	watchOutput string
+}
+
+func (c *changeCollector) Logf(msg string, args ...any) {
```

Comment:
are we actually log through collector?

Reply:
> are we actually log through collector?

Yeah, that is what the watch command does. It reports as it goes. I could do a separate change to revisit that. Or if you're his is controversial I can kick this or from the release. Jim has been waiting a few weeks now tho

Reply:
Please reopen if you want me to change something